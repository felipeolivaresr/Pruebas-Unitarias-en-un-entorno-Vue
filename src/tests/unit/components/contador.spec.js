import { shallowMount } from '@vue/test-utils'
import Contador from '@/components/Contador'


describe('Componente Contador.vue', () => {
    // test('Validación de match con el snapshot', () => {
    //Selección el Sujeto de pruebas 
    //////}),
    test('Validar el texto de un botón de enviar formulario', () => {
        const wrapper = shallowMount(Contador)
        const botonEnviar = wrapper.find('.boton-enviar')
        //Con este console.log comprobamos que estamos accediendo al texto del elemento 
        console.log(botonEnviar.text());
        expect(botonEnviar.text()).toBe("Enviar")
    })
})

