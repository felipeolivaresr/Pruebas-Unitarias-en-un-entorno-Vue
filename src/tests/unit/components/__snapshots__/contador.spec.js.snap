// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente Contador.vue Validaci√≥n de match con el snapshot 1`] = `
"<h2>Contador</h2>
<p class="counter">1</p>
<div><button class="btn primary incrementar">+1</button><button class="btn warning disminuir">-1</button></div>"
`;
